website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,123 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [BigQuery-specific configuration](/reference/resource-configs/bigquery-configs).
 
+### Optional configurations
+
+In BigQuery, optional configurations let you tailor settings for tasks such as query priority, dataset location, job timeout, and more. These options give you greater control over how BigQuery functions behind the scenes to meet your requirements.
+
+To customize your optional configurations in dbt Cloud:
+
+1. Click your name at the bottom left-hand side bar menu in dbt Cloud
+2. Select **Your profile** from the menu
+3. From there, click **Projects** and select your BigQuery project
+5. Go to **Development Connection** and select BigQuery
+6. Click **Edit** and then scroll down to **Optional settings**
+
+<Lightbox src="/img/bigquery/bigquery-optional-config.png" width="70%" title="BigQuery optional configuration"/>
+
+The following are the optional configurations you can set in dbt Cloud:
+
+| Configuration    | <div style={{width:'300px'}}>Information</div>   | Type    | Example            |
```

Comment:
added this to fix the table width bc it was looking weird 
<img width="749" alt="Screenshot 2024-10-28 at 15 58 41" src="https://github.com/user-attachments/assets/a09e3cb2-7c12-44c0-9e91-7b950b734f7d">